app/api/auth.py

```diff
@@ -0,0 +1,141 @@
+from flask import Blueprint, abort
+from flask.json import jsonify, request
+from flask_jwt_extended import create_access_token
+from app.api.utils import is_json_data, unique_id, hash_pwd, verify_pwd
+from .token import access_token
+from .. import db
+
+auth = Blueprint('auth', __name__)
+
+# manager signup auth route
+@auth.route('/manager/signup', methods=['POST'])
+def manager_signup():
+    is_json_data()
+    
+    # Get the values from json
+    manager_login = request.json.get("manager_login", None)
+    name = request.json.get("name", None)
```

Comment:
will be need more info for user registration and signup login will be same for all